src/main/java/io/strimzi/kafka/bridge/quarkus/RestBridge.java

```diff
@@ -357,7 +374,7 @@ private RestSinkBridgeEndpoint<byte[], byte[]> getRestSinkBridgeEndpoint(String
      * It creates and returns a new source endpoint, if the HTTP request comes from a new HTTP connection.
      *
      * @param routingContext RoutingContext instance used for getting the HTTP connection and attaching a close handler to it
-     * @param contentType Content-Type header from the HTTP request to be mapped to the corresponding embedded content type
+     * @param contentType    Content-Type header from the HTTP request to be mapped to the corresponding embedded content type
```

Comment:
why this change?

Reply:
I guess we can keep these changes since they align the descriptions of the parameter? We follow this pattern in the operator also.

Reply:
I would avoid this for now. Let's keep less changes and focus on what is the PR.

Reply:
Okay, let me revert it